lib/teleterm/teleterm_test.go

```diff
@@ -25,16 +25,18 @@ import (
 	"time"
 
 	"github.com/gravitational/teleport/lib/teleterm"
-
 	"github.com/stretchr/testify/require"
 )
 
 func TestStart(t *testing.T) {
```

Comment:
This can be done in a separate PR I think, but could you add a test which starts teleterm over TCP?

It'll be very similar to `TestStart` and [`TestGatewayStart`](https://github.com/gravitational/teleport/blob/08dcdcd27b54baf07bd939ed84ff05e75d0f0ac8/lib/teleterm/gateway/gateway_test.go#L49). A check which blocks until a server accepts connections on a TCP port is [almost identical](https://github.com/gravitational/teleport/blob/08dcdcd27b54baf07bd939ed84ff05e75d0f0ac8/lib/teleterm/gatewaytest/helpers.go#L31-L52) to `blockUntilServerAcceptsConnections` in this file. The main difference will be that in that function I linked to we expect `conn.Read(out)` to return `EOF` due to how ALPN proxies work but here you'll be able to do the same thing with `conn` that `blockUntilServerAcceptsConnections` does.

Reply:
Yeah, I will add it in a separate PR.